esx_service/utils/auth_data.py

```diff
@@ -44,6 +47,34 @@
 
 UPGRADE_README = "https://github.com/vmware/docker-volume-vsphere/blob/master/docs/misc/UpgradeFrom_Pre0.11.1.md"
 
+
+class DBMode(object):
```

Comment:
Sure. It can also be done with dictionaries, enums, classes , sets and what's not. All of them will work fine.  I do not see any value in changing the current implementation though.